packages/web-react/src/components/Dropdown/stories/Dropdown.tsx

```diff
@@ -0,0 +1,50 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+import { ComponentStory } from '@storybook/react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context/IconsContext';
+import { Icon } from '../../Icon';
+import Dropdown from '../Dropdown';
+import { SpiritDropdownProps, DropdownRenderProps } from '../../../types';
+// import { Button } from '../../Button';
+
+const Story: ComponentStory<typeof Dropdown> = (args: SpiritDropdownProps) => (
+  <IconsProvider value={icons}>
+    <Dropdown {...args} />
```

Comment:
Looks like the `id` attribute is missing from the dropdown menu itself:

![obrazek](https://user-images.githubusercontent.com/5614085/199211800-296c7a16-8e17-4fe9-97c7-ee3daa2fc424.png)

Reply:
Fixed